content/guides/guides/debugging.md

```diff
@@ -25,16 +25,28 @@ browser makes available to you, like `document`, `window`, and `debugger`.
 Based on those statements, you might be tempted to throw a `debugger` into your
 test, like so:
 
+:::cypress-visit-mount-test-example
```

Comment:
There are more code examples with `cy.visit()` as well as references to `cy.visit()` in the content area that should also probably be updated.

Reply:
👍  Adding additional tabbed examples and will pick up the content changes in UNIFY-522.